ibm/data_source_ibm_account.go

```diff
@@ -34,6 +58,26 @@ func dataSourceIBMAccountRead(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return fmt.Errorf("Error retrieving organisation: %s", err)
 	}
+
+	accountv1Client, err := meta.(ClientSession).BluemixAcccountv1API()
+	if err != nil {
+		return err
+	}
+	accountUsers, err := accountv1Client.Accounts().GetAccountUsers(account.GUID)
+	if err != nil {
+		return fmt.Errorf("Error retrieving users in account: %s", err)
+	}
+	accountUsersMap := make([]map[string]interface{}, 0, len(accountUsers))
+	for _, user := range accountUsers {
+		accountUser := make(map[string]interface{})
+		accountUser["id"] = user.Id
+		accountUser["email"] = user.Email
+		accountUser["state"] = user.State
+		accountUser["role"] = user.Role
+		accountUsersMap = append(accountUsersMap, accountUser)
```

Comment:
accountUsersMap[i] = accountUser would be cleaner to use I guess once you don't ignore the index in the for loop